eng/versioning/version_client_java_files.txt

```diff
@@ -4,10 +4,6 @@
 
 # The file format here should be the relative path from the root of the azure-sdk-for-java
 sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ClientConstants.java
-sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/AzureKeyVaultConfiguration.java
-sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/AzureKeyVaultConfiguration.java
-sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/AzureKeyVaultConfiguration.java
-sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BuilderHelper.java
```

Comment:
@JimSuplizio  added these files in for some reason. I am sure you may have spoken to him before removing them.

Reply:
Yes. Need to confirm with them. There are comments above, so I merge it into the PR as a fix to the issue listed in file.

Reply:
@hemanttanwar the reason for this list of files with versions is because of https://github.com/Azure/azure-sdk-for-java/issues/3141. If @sima-zhu is removing files with hard coded versions then I'm totally OK with it :)